web/modules/custom/dpl_event/src/Services/EventRestMapper.php

```diff
@@ -83,6 +85,51 @@ public function getResponse(EventInstance $event_instance): EventsGET200Response
     ]);
   }
 
+  /**
+   * Getting associated branches.
+   *
+   * @return ?string[]
+   *  The translated branch labels.
+   */
+  private function getBranches(): ?array {
+    $return = [];
+
+    /** @var \Drupal\node\NodeInterface[] $branches */
+    $branches = $this->event->get('branch')->referencedEntities();
+
+    foreach ($branches as $branch) {
+      $label = $branch->label();
+
+      if ($label instanceof TranslatableMarkup) {
+        $return[] = $label->render();
+      }
+      elseif (is_string($label)) {
+        $return[] = $label;
+      }
```

Comment:
Consider using [`getTitle()`](https://github.com/drupal/drupal/blob/10.1.x/core/modules/node/src/NodeInterface.php#L54-L61) instead of `label()`.

`getTitle()` seems to have a nice and simple signature. Having looked through the code I wonder why you use `label()`.